lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -146,6 +136,28 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
             });
     }
 
+    private async getDestinationFolderId() {
```

Comment:
can we name it getDestinationFolderNodeId?

Reply:
Updated, Thanks